### PR TITLE
chore: bump package versions

### PR DESCRIPTION
**Pull request by Benehiko**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,15 +30,14 @@
...
     "cookie-parser": "1.4.6",
     "csurf": "1.11.0",
     "express": "4.18.2",
     "express-handlebars": "6.0.7",
-    "express-jwt": "8.4.0",
-    "express-winston": "4.2.0",
+    "express-jwt": "8.4.1",
     "jwks-rsa": "3.0.1",
-    "winston": "3.8.2"
+    "express-winston": "4.2.0"
```
Comment (jonas-jonas) [inline package.json:40]: Why does the import not need to change?

**Review by jonas-jonas — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Benehiko) [reply package.json:40]: https://github.com/Marak/colors.js/issues/285

Comment (Benehiko) [reply package.json:40]: afaik, winston is not needed explicitly since we import `express-winston`

Comment (jonas-jonas) [reply package.json:40]: Ahh, yes, makes sense...

Comment (Benehiko) [reply package.json:40]: i'll update it

**Pull request merged**